docs/part-4-labeling-the-data-and-retrain/chapter-18-setup-label-studio.md

```diff
@@ -1,6 +1,204 @@
+wget https://github.com/swiss-ai-center/a-guide-to-mlops/archive/refs/heads/extra-data.zip -O extra-data.zip
+```
+
+The downloaded archive must be decompressed and renamed:
+
+```sh title="Execute the following command(s) in a terminal"
+# Extract the dataset
+unzip extra-data.zip
+
+# Rename to the folder to `extra-data`
+mv a-guide-to-mlops-extra-data/ extra-data/
+
+# Remove the archive and the directory
+rm extra-data.zip
+```
+
+Finally, add the `extra-data` folder to the `.gitignore` file:
+
+```sh title="Execute the following command(s) in a terminal"
+# Add the `extra-data` folder to the `.gitignore` file
+echo "\n# Label Studio\nextra-data/" >> .gitignore
+```
+
+Check the differences with Git to validate the changes:
+
+```sh title="Execute the following command(s) in a terminal"
+# Show the differences with Git
+git diff requirements.txt
+```
+
+The output should be similar to this:
+
+```diff
+diff --git a/.gitignore b/.gitignore
+index dee3012..cbfa93b 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -11,6 +11,3 @@ dvc_plots
+ 
+ # DVC will add new files after this line
+ /model
++
++# Label Studio
++extra-data/
+```
+
+### Install Label Studio
+
+Next, we will install Label Studio in our environment. Add the main
+`label-studio` dependency to the `requirements.txt` file:
+
+```txt title="requirements.txt" hl_lines="7"
+tensorflow==2.17.0
+matplotlib==3.9.2
+pyyaml==6.0.2
+dvc[gs]==3.53.2
+bentoml==1.3.7
+pillow==10.4.0
+label-studio==1.13.1
+```
+
+Check the differences with Git to validate the changes:
+
+```sh title="Execute the following command(s) in a terminal"
+# Show the differences with Git
+git diff requirements.txt
+```
+
+The output should be similar to this:
+
+```diff
+diff --git a/requirements.txt b/requirements.txt
+index 4b8d3d9..d584cca 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -4,3 +4,4 @@ pyyaml==6.0.2
+ dvc[gs]==3.53.2
+ bentoml==1.3.7
+ pillow==10.4.0
++label-studio==1.13.1
+```
+
+Install the package and update the freeze file.
+
+!!! warning
+
+    Prior to running any pip commands, it is crucial to ensure the virtual
+    environment is activated to avoid potential conflicts with system-wide Python
+    packages.
+
+    To check its status, simply run `pip -V`. If the virtual environment is active,
+    the output will show the path to the virtual environment's Python executable. If
+    it is not, you can activate it with `source .venv/bin/activate`.
+
+```sh title="Execute the following command(s) in a terminal"
+# Install the dependencies
+pip install --requirement requirements.txt
+
+# Freeze the dependencies
+pip freeze --local --all > requirements-freeze.txt
+```
+
+### Start Label Studio
+
+You can now start label studio with the following command:
+
+```sh title="Execute the following command(s) in a terminal"
+label-studio start
+```
+
+Label Studio will start on <http://localhost:8080>. Open the URL in your browser
+and sign up for an account.
```

Comment:
Explain the account creation is completely offline, and not related to any online service/teh enterprise offer of LabelStudio. This is only done to create an ID locally.